internal/dataplane/kongstate/kongstate.go

```diff
@@ -455,16 +455,16 @@ func (ks *KongState) FillPlugins(
 // that supports the FillID method (these are Service, Route, Consumer and Consumer
 // Group). It makes their IDs deterministic, enabling their correct identification
 // in external systems (e.g. Konnect Analytics).
-func (ks *KongState) FillIDs(logger logr.Logger) {
+func (ks *KongState) FillIDs(logger logr.Logger, workspace string) {
```

Comment:
Could you add a sentence of explenation about the parameter `workspace` in the docs for this function?

Reply:
Done.